client/components/ProjectTimeRequired.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styled from 'styled-components/native';
+import { FontAwesome5 } from '@expo/vector-icons';
+
+
+const ProjectTimeRequired = () => {
+
+    const TimeView = styled.View`
+    display: flex;
+    flex-direction: row;
+    margin-top: 7px;
+    `
+
+    const TimeText = styled.Text`
+    margin-left: 13px;
+
```

Comment:
styled components should be outside of the component definition as they need to be static